clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/ops/servergroup/AbstractEnableDisableKubernetesAtomicOperation.groovy

```diff
@@ -151,4 +158,22 @@ abstract class AbstractEnableDisableKubernetesAtomicOperation implements AtomicO
 
     null // Return nothing from void
   }
+
+  Boolean checkControllerType() {
```

Comment:
This should probably be something more descriptive, like "supportsEnableDisable"